indico/modules/rb/client/js/common/bookings/BookingEditForm.jsx

```diff
@@ -47,6 +47,17 @@ class BookingEditForm extends React.Component {
     hideOptions: {},
   };
 
+  showWarning = (initialValues, values) => {
```

Comment:
Let's find a better name for this. `shouldShowApprovalWarning` or something like that